modules/mobile/config/routes.rb

```diff
@@ -18,6 +18,7 @@
     patch '/appointments/check-in/demographics', to: 'check_in_demographics#update'
     post '/appointment', to: 'appointments#create'
     get '/appointments/check-in/demographics', to: 'check_in_demographics#show'
+    get 'awards', to: 'awards#index'
```

Comment:
Starting a no leading slash trend? I like it.

Reply:
Don't worry it's changed

Reply:
I have no idea why we use the leading slashes. Doesn't seem to impact anything.